docs/index.html

```diff
@@ -1,150 +1,196 @@
-<!DOCTYPE HTML>
+<!DOCTYPE html>

```

Comment:
Most of the changes in this file are formatting, with the exception of the new "viewer" section and JS/CSS script additions.